waspc/src/Wasp/AI/GenerateNewProject/Common/Prompts.hs

```diff
@@ -93,7 +95,7 @@ waspFileExample =
           component: import Login from "@client/pages/auth/Login.jsx"
         }
 
-        route DashboardRoute { path: "/", to: Dashboard }
+        route DashboardRoute { path: "/", to: DashboardPage }
```

Comment:
Ouf, we had a mistake here, did we? Nice catch! This must have confused it to some degree.